src/rgw/rgw_rados.cc

```diff
@@ -8505,8 +8505,11 @@ int RGWRados::open_bucket_index(const RGWBucketInfo& bucket_info, librados::IoCt
 {
   const rgw_bucket& bucket = bucket_info.bucket;
   int r = open_bucket_index_ctx(bucket_info, index_ctx);
-  if (r < 0)
+  if (r < 0) {
+    ldout(cct, 20) << "open_bucket_index: open_bucket_index_ctx() returned "
```

Comment:
@rzarzynski Is there specific purpose of prefixing `open_bucket_index:` in message.
Main_func:called_func sort of?